spec/search_builders/curation_concerns/admin_set_search_builder_spec.rb

```diff
@@ -9,7 +9,7 @@
     before { builder.filter_models(solr_params) }
 
     it 'adds AdminSet to query' do
-      expect(solr_params[:fq].first).to include('{!field f=has_model_ssim}AdminSet')
```

Comment:
These generate functionally equivalent queryies.  "terms" is more correct for our purposes because it's not a fielded search.